.goreleaser.yaml

```diff
@@ -90,7 +90,15 @@ nfpms:
         packager: deb
 
 archives:
-  - name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+  - name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else if eq .Arch "windows" }}Windows
+      {{- else if eq .Arch "linux" }}Linux
+      {{- else if eq .Arch "darwin" }}Darwin
```

Comment:
It looks like based on the docs you linked you don't need overrides for these?

Reply:
You're right, these are not needed.